client/src/api/teams/index.ts

```diff
@@ -1,7 +1,8 @@
-import { Response } from '../types';
-import { get, buildEndpoint } from '../builders';
+import { ApiResponseBase, Response } from '../types';
+import { get, buildEndpoint, post, put } from '../builders';
 
 import { TeamsResponse } from './types';
+import { ITeam } from '../../../../common';
```

Comment:
Please fix this import and fix the lint errors. Package imports like `ssw-common`, `react`, `semantic-ui-react` should all go at the top of a file.

Reply:
And by fix this import I mean do not use `../../../../common` to get the ITeam. Use `ssw-common`